.azure-pipelines/pipelines.yml

```diff
@@ -73,6 +73,11 @@ stages:
 # Presubmit/default
 - ${{ if eq(variables.pipelineDefault, true) }}:
   - template: stages.yml
+    parameters:
```

Comment:
PR LGTM when moved to non-WiP (without this unrelated change which I assume goes away then).

Reply:
yep - its failing proto_format (which was kinda expected) - so this was there to bypass prechecks